lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
hehe, thank you for pointing that out @gmurphey. We definitely want to have things go through the `ui` object if at all possible...